server/service/src/sync/api_v6/mod.rs

```diff
@@ -157,19 +159,22 @@ pub struct SyncPullRequestV6 {
     pub(crate) batch_size: u32,
     pub(crate) sync_v5_settings: SyncApiSettings,
     pub(crate) is_initialised: bool,
+    pub(crate) sync_v6_version: u32,
 }
 
 #[derive(Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct SyncPushRequestV6 {
     pub(crate) batch: SyncBatchV6,
     pub(crate) sync_v5_settings: SyncApiSettings,
+    pub(crate) sync_v6_version: u32,
 }
 
 #[derive(Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct SiteStatusRequestV6 {
     pub(crate) sync_v5_settings: SyncApiSettings,
+    pub(crate) sync_v6_version: u32,
```

Comment:
It's slightly more comiplex, but I think for v7 api (when we sync to omsuplly central with omsupply central credentials), we would want something like the following (example doesn't annotate serde etc..)

```
stuct RequestV7<T> {
   common: // credentials, version etc..
   data: <T>
}

struct PushPayload {
   batch: SyncBatchv7
}

type PushV7 =  Request<PushPayload>
```